### ISSUE TITLE
"aud" for access token returns as "00000003-0000-0000-c000-000000000000" but should be a specific string

### ISSUE THREAD

**Issue by adriansalgadoa** [AzureAD/microsoft-authentication-library-for-js#5672]
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

2.32.2

### Wrapper Library

MSAL React (@azure/msal-react)

### Wrapper Library Version

1.5.2

### Public or Confidential Client?

Public

### Description

When getting the response from `instance.acquireTokenSilent`, the token has an unexpected default AUD. It should be something along the lines of "bridgeit", but I don't see any documentation on how to pass this on the main config

I've seen on other implementations this is passed as Audience or Resource id, but can't find any reference to this

### MSAL Configuration

```javascript
export const msalConfig: Configuration = {
  auth: {
    clientId: import.meta.env.VITE_MICROSOFT_CLIENT_ID,
    authority: import.meta.env.VITE_MICROSOFT_AUTHORITY,
    redirectUri: 'http://localhost:5173'
  },
  cache: {
    cacheLocation: 'localStorage',
    storeAuthStateInCookie: true 
  }
};
```

### Relevant Code Snippets

```javascript
Token received from response

eyJ0eXAiOiJKV1QiLCJub25jZSI6IndqQ19xeF9Na29aNmI3T0oxLWNKNWJVZHRnRThfbXJoZm1PM0VDWWl3SDAiLCJhbGciOiJSUzI1NiIsIng1dCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyIsImtpZCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyJ9.eyJhdWQiOiIwMDAwMDAwMy0wMDAwLTAwMDAtYzAwMC0wMDAwMDAwMDAwMDAiLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC9kMzgwMzVkYi1mZTA5LTQ5OTMtYWVhYi1hYTk3MDIyZGIxMWUvIiwiaWF0IjoxNjc1OTMyNDExLCJuYmYiOjE2NzU5MzI0MTEsImV4cCI6MTY3NTkzNjUzOCwiYWNjdCI6MCwiYWNyIjoiMSIsImFpbyI6IkFWUUFxLzhUQUFBQS94N2l3TUtpeGh4MzRITGZnNEJoUTNVSnlsZWJOTlozQncvcVNodVRkeElFT0JMY01xNjJUdDZUc01GYkE1YWZGMHgzTEpYOUhIUTAyWVc5WEZDOHNjM05wSTBhSmJLQ3UyRDkvaHo0Yi80PSIsImFtciI6WyJwd2QiLCJtZmEiXSwiYXBwX2Rpc3BsYXluYW1lIjoiTXlFbXBhY3QiLCJhcHBpZCI6ImM1ZDcxZmFlLTkxY2QtNGNjYy1hZGVmLWYyNzRkNWI3NzIyOSIsImFwcGlkYWNyIjoiMCIsImZhbWlseV9uYW1lIjoiU2FsZ2FkbyIsImdpdmVuX25hbWUiOiJBZHJpYW4iLCJpZHR5cCI6InVzZXIiLCJpcGFkZHIiOiIxMjguNzcuMjEwLjEyNiIsIm5hbWUiOiJBZHJpYW4gU2FsZ2FkbyIsIm9pZCI6IjNjNDM4ZTU0LTE3OTctNDE0ZS1iZjliLTc1YTZlOGU3MmZiZiIsInBsYXRmIjoiMyIsInB1aWQiOiIxMDAzMjAwMjVBRDY2MDcyIiwicmgiOiIwLkFVZ0EyeldBMHduLWswbXVxNnFYQWkyeEhnTUFBQUFBQUFBQXdBQUFBQUFBQUFCSUFPdy4iLCJzY3AiOiJBbGxTaXRlcy5SZWFkIERpcmVjdG9yeS5BY2Nlc3NBc1VzZXIuQWxsIE15RmlsZXMuUmVhZCBVc2VyLlJlYWQgVXNlci5SZWFkQmFzaWMuQWxsIFVzZXIuUmVhZFdyaXRlIHByb2ZpbGUgb3BlbmlkIGVtYWlsIiwic3ViIjoiU3BfOW5NRGdadTA5LU9aY1RYbjY1bEh4TmJVQ3g1Z09lUVo0T2dwNDlrayIsInRlbmFudF9yZWdpb25fc2NvcGUiOiJFVSIsInRpZCI6ImQzODAzNWRiLWZlMDktNDk5My1hZWFiLWFhOTcwMjJkYjExZSIsInVuaXF1ZV9uYW1lIjoiYWRyaWFuLnNhbGdhZG9AZW1wYWN0LmFwcCIsInVwbiI6ImFkcmlhbi5zYWxnYWRvQGVtcGFjdC5hcHAiLCJ1dGkiOiJmQ0NKekhsSE9FeXp3MlNIWm5meEFBIiwidmVyIjoiMS4wIiwid2lkcyI6WyJiNzlmYmY0ZC0zZWY5LTQ2ODktODE0My03NmIxOTRlODU1MDkiXSwieG1zX3N0Ijp7InN1YiI6Im9MVldCT0RvTFBrVUFBbnRZLVJKVVA0ZkluYVRydC12eXR6Y1lHd2JfR0UifSwieG1zX3RjZHQiOjEzOTA5OTM5NTMsInhtc190ZGJyIjoiRVUifQ.CrPzj-wOQACAmYop0sH6if-LY5hTfcwTN64Wo5rBfUy2Q6lBejgQDKzbIBfn-vyQrw5YBkTG8gh6ZienHiNC64TzUooX-LyF6sYRZmXnhIfXRk1q1MjyLelqwMKWjdOnZADlFB63TU-7-l7LQh2cpBL6o9TCjgfULWDsR6bC5fkTflLwJz1G6BLe-puLn-HjL6K1TBfhmnieCN-JkvEuoW9jxviGBf7mbKimAfB4cTqBjHaC231dWa5dwrR_XnyuOIWNiUzpMNsB56PXdZX188XtsgwP7VEIz9Y5DmW7Q_Hbj8LxHj90zwStT-ws_OgsZR4qMhuYygilw_gQiVJRVw
```

### Identity Provider

Azure AD / MSA

### Source

External (Customer)

**Comment by sameerag** [comment]
@adriansalgadoa Can you share your request object? "aud" should be dependent on the corresponding token request.
`The aud claim in a token indicates the resource that the token is intended for (its audience). Clients use the token but shouldn't understand or attempt to parse it. Resources accept the token.` More details [here](https://learn.microsoft.com/en-us/azure/active-directory/develop/access-tokens#token-ownership).

So is your token request for graph?  The `aud` "00000003-0000-0000-c000-000000000000" is GUID of the graph API.

**Comment by adriansalgadoa** [comment]
@sameerag I think it should be this

```
export const loginRequest: RedirectRequest = {
  scopes: ['User.Read']
};
```

But right now I'm getting CORS issues for some reason, and can't clearly now why.

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://login.microsoftonline.com/d38035db-fe09-4993-aeab-aa97022db11e/oauth2/v2.0/token. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 200.
```

**Comment by adriansalgadoa** [comment]
Ok so, we worked it out! I was using the wrong scope, it works if I use `https://bridgeit.dk/bmp/user_impersonation`, and the CORS issue we managed to fix it by changing our localhost, apparently there's a localhost bug where it wont respect the port. We had a few localhost:4200 in Web platforms and our SPA was setup on localhost:5137, but if we change it to 4200 it worked!

So yeah, not sure if I can report that bug to you somewhere.
